src/components/Sidebar/components/NavList/NavList.js

```diff
@@ -25,7 +27,15 @@ const baseStyles = () => css`
   width: 100%;
 `;
 
-const NavList = styled('ul')(baseStyles);
+const StyledList = styled('ul')(baseStyles);
+
+const TRACKING_ELEMENT = 'navList';
```

Comment:
Would it make sense to add these constants to a common constants file instead of in each component?

Reply:
Also asking again about naming conventions — I think we should use kebab case everywhere.

Reply:
These values where defined by the analysts

Reply:
Ugh, do they have/follow a naming convention? 🙈